scm-ui/ui-webapp/src/repos/components/form/repositoryValidation.ts

```diff
@@ -25,10 +25,11 @@
 import { validation } from "@scm-manager/ui-components";
 
 const nameRegex = /(?!^\.\.$)(?!^\.$)(?!.*[.]git$)(?!.*[\\\[\]])^[A-Za-z0-9\.][A-Za-z0-9\.\-_]*$/;
-const namespaceExceptionsRegex = /^(([0-9]{1,3})|(create))$/;
+const namespaceExceptionsRegexCreate = /^(([0-9]{1,3})|(create))$/;
+const namespaceExceptionsRegexImport = /^(([0-9]{1,3})|(import))$/;
```

Comment:
Why not `/^(([0-9]{1,3})|(create)|(import))$/` instead of two separate regexes with the same first case?